ui/app/src/components/SiteDashboard/SiteDashboard.tsx

```diff
@@ -143,6 +144,17 @@ export function Dashboard(props: DashboardProps) {
             <DashboardSkeleton />
           )}
         </Suspense>
+        {/* Displays on desktop - inside grid, below the last rendered dashlet */}
+        <Grid
+          item
+          md={12}
+          sx={{
+            display: desktopScreen ? 'flex' : 'none',
+            flexWrap: 'wrap'
+          }}
+        >
+          <IconGuideDashlet />
+        </Grid>
```

Comment:
Wrap both of these in an `if` based on `desktopScreen` so they only render once on their respective screen size.